### ISSUE TITLE
Support multiple -set files with base configuration

### ISSUE THREAD

**Issue by sanhozay** [IAHM-COL/727-230#11]
For development of multiple variants with multiple -set.xml files, common configuration needs to move to a -base.xml or equivalent, with the Yasim and JSB variants importing it.

**Comment by IAHM-COL** [comment]
@sanhozay 
Correct. 
Also see related issue #3

**Comment by JWocky** [comment]
Agreed, but give me a little time till the A/P works then I can move it also to the -base

**Comment by sanhozay** [comment]
We can leave the autopilot selection up in the -set files if you would like? Then you can have a different autopilot for JSB and Yasim if need be. Or just move it down into the base.

**Comment by JWocky** [comment]
I would like to move it down to base, but I have still some issues with the new one. On the other hand, the sound, a typical item for the -base sits still in the JSB-set. So, I go and play a bit with the A/P around till I get it right and hopefully, we can Friday change to -base with AP, sound, splash and all the other shebang if that is okay by you.

**Comment by FGDATA** [comment]
I added the sound to both sets YAsim and JSBsim myself. That's fixed.
I also transferred stereos to mono

Best
IH-COL

**Comment by FGDATA** [comment]
Yet, I totally agree with G-SHOZ in doing this "base file " earlier
So, I am all up for his updating the sound into base and redirect the
respective sets, again

About AP, I guess we need to keep them non-base until we get a "stable
one", on which JWocky is tinkering.

Right?
IH-COL

**Comment by JWocky** [comment]
Correct, IAHM-COL. I go on it as soon as I get my forum posts read.
